shared/upload/utils.py

```diff
@@ -0,0 +1,56 @@
+from datetime import timedelta
+from enum import Enum
+
+from django.db.models import Q
+from django.utils import timezone
+
+from shared.django_apps.codecov_auth.models import Owner, TrialStatus
+from shared.django_apps.core.models import Commit, Repository
+from shared.django_apps.reports.models import ReportSession, ReportType
+from shared.django_apps.user_measurements.models import UserMeasurement
+from shared.plan.service import PlanService
+
+
+class UploaderType(Enum):
+    LEGACY = "legacy"
+    CLI = "cli"
+
+
+def query_monthly_coverage_measurements(plan_service: PlanService) -> int:
+    owner = plan_service.current_org
+    queryset = UserMeasurement.objects.filter(
+        owner=owner,
+        private_repo=True,
+        created_at__gte=timezone.now() - timedelta(days=30),
+        report_type="coverage",
```

Comment:
maybe this can be `ReportType.COVERAGE.value`

Reply:
yeap good callout, pushing in a bit